model/mention_map.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type UserMentionMap map[string]string
+type ChannelMentionMap map[string]string
+
+const (
+	UserMentionsKey       = "user_mentions"
```

Comment:
Would it be worth not exporting these from the package, since it seems like we don't need them yet anywhere else?

Reply:
Makes sense!

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/13762/commits/552c67e6d459e98b646fd149c792a2bb0eb7486d. We can make them public in the future if we ended up needing them.